cli/azd/internal/repository/infra_confirm.go

```diff
@@ -1,8 +1,10 @@
 package repository
```

Comment:
Can we also inspect any docker-compose.y*ml files to see if there's a port mapping there as well?

Reply:
@jongio I'm wondering if my understanding is correct: would docker-compose describe the application workload, and not a given service? If docker-compose is referring to the app workload, I think we may need to address this separately as "init from docker-compose.yml".

Reply:
I was referring to checking the services in docker-compose.yml to see if any have port mappings we need to address.  They are usually the same but sometimes different.  It's fine to consider this later.